src/app/search/search-results/search-results.component.html

```diff
@@ -56,25 +65,34 @@
     </ng-template>
     <div class="mat-tab-content">
       <div *ngIf="workflowTagCloudData">
-        <button type="button" class="btn btn-info tagCloud-btn workflow" (click)="clickTagCloudBtn('workflow')">
-          <i [class]="(showWorkflowTagCloud$ | async) ? 'fa fa-angle-double-up' : 'fa fa-angle-double-down'" aria-hidden="true"></i>
-          <span>Tag Cloud</span>
-        </button>
-        <div class="tagCloud workflow" *ngIf="showWorkflowTagCloud$ | async">
-          <angular-tag-cloud
-            [data]="workflowTagCloudData"
-            [width]="options.width"
-            [height]="options.height"
-            [overflow]="options.overflow"
-            (clicked)="tagClicked($event)"
-          >
-          </angular-tag-cloud>
+        <div fxLayout="row wrap" fxLayoutAlign="space-between center">
+          <span fxFlex="65%">
+            <p>A <b>Workflow</b> can use multiple containers and exectues multiple actions or steps, outlined by a descriptor</p>
+          </span>
+          <div class="tag-cloud-dropdown">
```

Comment:
This block looks the same as the block starting on line 30. Good candidate for a component. Your call; don't want to over-engineer either.

Reply:
I'm OK leaving this as is, but you have a thumbs up, but didn't change it. Just want to make sure that's your intent, and that you didn't overlook it.

Reply:
Yup, chose not to change it